Task3

Emilen096 opened the pull request.

Emilen096 committed:
Task

Emilen096 committed:
Task

Emilen096 committed:
Add files via upload

BalakinaEP reviewed (changes requested):
Тесты

Emilen096 committed:
Add files via upload

BalakinaEP reviewed (changes requested):
gogle tests или unit tests

Emilen096 committed:
Errors in google tests

"GetPosition": не является членом "UniversityEmployee".
класс "UniversityEmployee" не содержит члена "GetPosition"
класс "UniversityEmployee" не содержит члена "GetPosition"
testing::internal::EqHelper<false>::Compare: функция не принимает 3 аргументов
"GetPosition": не является членом "UniversityEmployee".
testing::internal::EqHelper<false>::Compare: функция не принимает 3 аргументов
